packages/dd-trace/src/appsec/iast/vulnerability-reporter.js

```diff
@@ -95,6 +102,7 @@ function deduplicateVulnerabilities (vulnerabilities) {
 
 function start (config, _tracer) {
   deduplicationEnabled = config.iast.deduplicationEnabled
+  standaloneEnabled = !!config.appsec?.standalone?.enabled
```

Comment:
should I use here `standalone` module? It'd introduce a dependency between iast and appsec...

Reply:
iast directory is into `appsec` directory, it looks like there is a dependency by design.

Reply:
oki doki